aries-site/src/examples/templates/filtering/FilteringCards.js

```diff
@@ -0,0 +1,607 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import {
+  Anchor,
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CheckBoxGroup,
+  FormField,
+  Header,
+  Heading,
+  Grid,
+  Layer,
+  RangeSelector,
+  ResponsiveContext,
+  Stack,
+  Text,
+  TextInput,
+} from 'grommet';
+import { Filter, FormClose, Search } from 'grommet-icons';
+
+const allData = [
+  {
+    location: 'San Jose, CA',
+    hoursAvailable: 10,
+    image:
+      'https://images.unsplash.com/photo-1584704135557-d8bf7ca50eae?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    role: 'Engineer',
+    name: 'Eric Soderberg',
+    status: 'Online',
+  },
+  {
+    location: 'San Jose, CA',
+    hoursAvailable: 30,
+    image:
+      'https://images.unsplash.com/photo-1539037116277-4db20889f2d4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    role: 'Engineer',
+    name: 'Taylor Seamans',
+    status: 'Online',
+  },
+  {
+    location: 'Fort Collins, CO',
+    hoursAvailable: 25,
+    image:
+      'https://images.unsplash.com/photo-1503424886307-b090341d25d1?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1955&q=80',
+    role: 'Engineer',
+    name: 'Matthew Glissmann',
+    status: 'Offline',
+  },
+  {
+    location: 'Fort Collins, CO',
+    hoursAvailable: 5,
+    image:
+      'https://images.unsplash.com/photo-1501594907352-04cda38ebc29?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2389&q=80',
+    role: 'Designer',
+    name: 'Greg Furuiye',
+    status: 'Online',
+  },
+  {
+    location: 'San Jose, CA',
+    hoursAvailable: 25,
+    image:
+      'https://images.unsplash.com/photo-1534430480872-3498386e7856?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    role: 'Designer',
+    name: 'Vicky Avalos',
+    status: 'Offline',
+  },
+  {
+    location: 'Fort Collins, CO',
+    hoursAvailable: 12,
+    image:
+      'https://images.unsplash.com/photo-1584450150050-4b9bdbd51f68?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    role: 'Engineer',
+    name: 'Shimi Yacobi',
+    status: 'Online',
+  },
+];
+
+const StyledCard = styled(Card)`
+  transition: all 0.3s ease-in-out;
+  :focus,
+  :hover {
+    transform: scale(1.01, 1.01);
+  }
+`;
+
+const StyledButton = styled(Button)`
```

Comment:
Good point, yes it doesn't align with the button props of `default`, `secondary`, or `primary` .. I will reach out to Greg to see if we want to introduce this new pattern or rely on the default

Reply:
Update: Greg confirmed that for control buttons like this, we want to move in this design direction of border and no background fill. The easiest way I thought of leveraging the theme to do this would be to designate some name to this kind of button (maybe `control`) and then leverage the extend of the theme to apply the styling:
```
button: {
    extend: ({ control, theme }) =>
      css`
        ${control &&
        `border: 1px solid
    ${theme.global.colors.border[theme.dark ? 'dark' : 'light']};
  &:hover {
    background: transparent;
  }`}
      `,
  },
```
and implement via `<Button label="Label" control />`

However, this naming/approach feels like it warrants a discussion that includes Eric and Shimi. I'm proposing we leave as a styled component for the first pass then have that discussion next week. Thoughts?

Reply:
Agree to leaving as is and creating an issue to discuss the "control button" direction.

Reply:
Being tracked here: https://github.com/hpe-design/design-system/issues/1118